<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopperDialog</class>
 <widget class="QDialog" name="CopperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="registers_list_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <column>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>251</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QListWidget" name="copperlist_2">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Copperlist #2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>251</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QListWidget" name="copperlist_1">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Copperlist #1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <zorder>gridLayoutWidget</zorder>
  <zorder>registers_list_</zorder>
  <zorder>gridLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
